docs/how-to-use-proof.md

```diff
@@ -0,0 +1,31 @@
+# A Guide on How to Use Asset Proofs in Scalar DL
+
+This document sets out some guidelines for using Asset Proofs in Scalar DL.
+
+## What is an Asset Proof in Scalar DL?
+
+An Asset Proof in Scalar DL is a set of information about an asset record that is created along with the asset record. It is composed of the following items so that it can be treated as evidence of the creation of an asset record by a ledger.
+
+- The ID of an asset record
+- Age of the asset record
+- Nonce of the execution request that creates the asset record
+- A cryptographic hash of the asset record
+- The digital signature of the above four records 
+
+## The Benefits of Asset Proofs
+
+Since Asset Proofs are evidence at the time of execution by a ledger, it is hard for the leger to tamper data after the evidence is created because the proofs and the ledger states would be inconsistent.
+Thus, using and managing Asset Proofs appropriately could reduce the risk of tampering of data, so it could make Scalar DL more tamper-evident without creating additional computational resources such as ordering components.
```

Comment:
```suggestion
Thus, making use of Asset Proofs appropriately could reduce the risk of tampering of data, so it could make Scalar DL more tamper-evident without creating additional computational resources such as ordering components.
```